.github/workflows/create-release.yml

```diff
@@ -32,13 +32,15 @@ jobs:
           # The sed removes carriage returns so that the body is easier to parse later, and
           #   escapes backticks so that they are not executed as commands.
           PR_BODY=$(printf '%s' "$PR" | jq '.[0].body' | sed 's/\\r//g' | sed 's/`/\\`/g')
-          echo pr_body=${PR_BODY} >> $GITHUB_OUTPUT
+          echo "pr_body<<EOF" >> $GITHUB_ENV
```

Comment:
did changing from output to env fix this, or is this just what the practice is supposed to be?

I thought they were the same place

Reply:
Official documentation mentions multiline strings only with $GITHUB_ENV. I assume it's the desired way to pass multiline strings through the pipeline.
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings

Reply:
And yeah, both $GITHUB_OUTPUT and $GITHUB_ENV are paths to some magic files, but it's different files.